mxcubeweb/core/components/user/usermanager.py

```diff
@@ -91,7 +105,12 @@ def set_operator(self, username):
 
         return user
 
-    def update_active_users(self):
+    def update_active_users(self) -> None:
+        """
+        Check if any user have been inactive for a period longer than the
+        session lifetime. If so, deactivate the user in datastore and emit
+        the relvant signals `userChanged` and `observersChanged` to the client.
+        """
```

Comment:
About docstrings:
* the first sentence must be a summary that must fit on one line including the opening set of 3 double quotes
* It is reStructuredText syntax, not Markdown, so we need two backticks

```suggestion
        """Check if any user have been inactive for longer than session lifetime.
        
        If so, deactivate the user in datastore and emit
        the relevant signals ``userChanged`` and ``observersChanged`` to the client.
        """
```

Reply:
ok